### ISSUE TITLE
readgeis fails

### ISSUE THREAD

**Issue by gbrammer** [spacetelescope/stsci.tools#59]
I'm trying to generate multi-extension FITS versions of WFPC2 calibration files and am having a problem with `stsci.tootls.readgeis`:

```python
In [2]: hdu = readgeis.readgeis('m3c1004nu.r4h')
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-f76cf6d45c73> in <module>()
----> 1 hdu = readgeis.readgeis('m3c1004nu.r4h')

/usr/local/share/python/anaconda3/envs/iraf27/lib/python2.7/site-packages/stsci.tools-3.4.11-py2.7.egg/stsci/tools/readgeis.pyc in readgeis(input)
    203     _shape = _naxis[1:]
    204     _shape.reverse()
--> 205     _code = fits.hdu.ImageHDU.NumCode[_bitpix]
    206     _bscale = phdr.get('BSCALE', 1)
    207     _bzero = phdr.get('BZERO', 0)

AttributeError: type object 'ImageHDU' has no attribute 'NumCode'
```

The "NumCode" bit appears to have been deprecated in `pyfits` some time ago.  Any suggestions for making this work?  

*Edit:*
`stsci.tools.convertwaiveredfits` does appear to work on the waivered fits calibration files provided by MAST.  However, the issue above still applies if someone tries to convert GEIS files as provided by CRDS, e.g., https://hst-crds.stsci.edu/bestrefs/.

**Comment by pllim** [comment]
@gbrammer , all the GEIS should have been converted to MEF a long time ago (at least in MAST). I am a little shocked that the reference files are still distributed in GEIS/wFITS formats. I will contact appropriate team and get back to you. Thank you for your patience.

In the meantime, your best bet right now is to use an older version of Astropy before they removed `NumCode` support.

**Comment by gbrammer** [comment]
Thanks, @pllim.  Yes, I was surprised, too.  The science data seem to be distributed in MEF format, but at least the flat-field file I mentioned above ('m3c1004nu_r4h.fits') I got with the same query seemed to be in waivered FITS.  There was a bit strange behavior with MAST in that if I clicked the "Deliver WFPC2 datasets in Waivered FITS format" box, I didn't get *any* calibration files with the delivery.  I know these issues are probably outside of the scope of the `stsci.tools` software, I just wanted to make a note of them here.
